docs/file_extras/tools/open_file_tool/README.md

```diff
@@ -9,36 +9,47 @@ from dotenv import load_dotenv
 
 from griptape.file_extras.tools.open_file_tool import OpenFileTool
 from griptape.structures import Agent
+from griptape.drivers import OpenAiChatPromptDriver
+
+import os
 
 load_dotenv()
 
+key = "OPENAI_API_KEY"
+if not (OPENAI_API_KEY := os.getenv(key)):
+    raise ValueError(f"{key} missing")
```

Comment:
`os.environ["OPENAI_API_KEY"]` will throw a `KeyError` if not found. If you'd prefer to have a custom implementation we should make a utility (`griptape.file_extras.utils.env_utils`) something like this:
```python
class MissingEnvironmentVariable(Exception):
    pass


def get_env_var(var_name):
    try:
        return os.environ[var_name]
    except KeyError:
        raise MissingEnvironmentVariable(f"{var_name} does not exist")
```